grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -235,25 +276,25 @@ def _filter(self, filter_p, force_unzip=False):
         return files
 
     def import_products(self, reproject=False, link=False, override=False):
-        args = {}
+        self._args = {}
         if link:
             module = "r.external"
-            args["flags"] = "o" if override else None
+            self._args["flags"] = "o" if override else None
         else:
-            args["memory"] = options["memory"]
+            self._args["memory"] = options["memory"]
             if reproject:
                 module = "r.import"
```

Comment:
For the sake of clarity and readability I would leave this as I've proposed it. It might get difficult to understand the script when implementing self._module since different modules are called depending on the users decision to reproject the data (either calling r.import or r.in.gdal), don't you think?

Reply:
This is fine by me